bitwarden_license/bit-web/src/app/secrets-manager/overview/overview.component.html

```diff
@@ -1,3 +1,40 @@
 <sm-header>
   <sm-new-menu></sm-new-menu>
 </sm-header>
+
+<sm-onboarding [title]="'getStarted' | i18n" *ngIf="completed$ | async as completed">
+  <sm-onboarding-task
+    [title]="'createServiceAccount' | i18n"
+    (click)="openServiceAccountDialog()"
+    icon="bwi-cli"
+    [completed]="completed.createServiceAccount"
+  >
+    <span
+      >{{ "downloadThe" | i18n }} <a bitLink routerLink="">{{ "smCLI" | i18n }}</a></span
+    >
```

Comment:
Assuming this doesn't mess up the spacing.

```suggestion
    <span>
      {{ "downloadThe" | i18n }} <a bitLink routerLink="">{{ "smCLI" | i18n }}</a>
    </span>
```